src/Bicep.Core/Parsing/Parser.cs

```diff
@@ -1,370 +1,184 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Globalization;
-using System.Linq;
-using Bicep.Core.Diagnostics;
-using Bicep.Core.Extensions;
-using Bicep.Core.Navigation;
-using Bicep.Core.Syntax;
-
-namespace Bicep.Core.Parsing
-{
-    public class Parser
-    {
-        private readonly TokenReader reader;
-
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Linq;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Extensions;
+using Bicep.Core.Navigation;
+using Bicep.Core.Syntax;
+
+namespace Bicep.Core.Parsing
+{
+    public class Parser
+    {
+        private readonly TokenReader reader;
+
```

Comment:
@shenglol - do you know why your PRs show large whitespace diffs? Are there line ending changes here? If so, any way to avoid this? I notice that when I modify code afterwards, I also get large whitespace diffs (presumably to change back to the previous format). It would be good if we can converge on / enforce a standard here.

Reply:
I suspect that the line endings of this file got normalized by someone's Visual Studio settings. The whitespace diffs showed up after I merged the main branch. I feel like we should put a .editorconfig file at the root level and add that as a VS solution item. I submit a PR to do that later.

Reply:
Created https://github.com/Azure/bicep/issues/1401 to track this.